coalib/coala.py

```diff
@@ -54,9 +54,11 @@ def main(debug=False):
         configure_logging(not args.no_color)
 
         if args.show_bears:
-            from coalib.settings.ConfigurationGathering import (
-                get_all_bears)
-            filtered_bears = get_all_bears()
+            from coalib.settings.ConfigurationGathering import get_all_bears
```

Comment:
unnecessary change. If it fits into a single line, please do a separate commit for it :+1:
(which is defnitely reasonable, having this in a single line reads way better)

Reply:
(I mean just the import line, not the `filtered_bears` thing :D)